### ISSUE TITLE
New release to enable TLS1.3 in downstream projects

### ISSUE THREAD

**Issue by micheelengronne** [arlimus/sslshake#12]
Hi,

can you create a new release to enable TLS1.3 in downstream projects.

The support was added via https://github.com/arlimus/sslshake/pull/11

We need the release to enable it in https://github.com/inspec/inspec/issues/4956

And then in https://github.com/dev-sec/ssl-baseline/pull/31

**Comment by marcIhm** [comment]
Yes, I would like to see a new release too. This Libary seems to be the only easy way to check for sslv2 and 3 without having to keep around an old binary.
